### PR TITLE
PR: Rename Path by Current Working Directory label option (Find in Files)

### LINKED ISSUE #4693: Change "Path" option to "Current working directory" in Find in Files
This is an easy one for you @andfoy. I refer to this "Path":

seleccion_003

This is necessary now that we're simplifying all options related to *Working directory*.

### PR DESCRIPTION
**Pull request by andfoy**
Fixes #4693

### CODE REVIEW

**Comment by ccordoba12** [comment]
@andfoy, please rebase this one onto `3.x`, you did it against master.

File: spyder/widgets/findinfiles.py [hunk 1]
```diff
@@ -259,7 +259,8 @@ def __init__(self, parent, search_text, search_text_regexp, search_path,
         # Layout 3
         hlayout3 = QHBoxLayout()
 
-        self.global_path_search = QRadioButton(_("Path"), self)
+        self.global_path_search = QRadioButton(_("Current Working "
+                                                 "Directory"), self)
```
Comment (ccordoba12) [inline spyder/widgets/findinfiles.py]: Please change this string to `Current working directory` (i.e. loose the capitals in `Working` and `Directory`)

**Comment by andfoy** [comment]
@ccordoba12 Something went wrong with my local branch while I was performing the rebase. I will open this PR from another branch

**Comment by ccordoba12** [comment]
Ok, no prob ;-)

**Pull request closed without merging**